README.md

```diff
@@ -74,14 +75,16 @@ The optional enhancement for this wave is to implement a simulation of the broad
     * Expect that the `change` parameter will be a number, positive OR negative.
     * Add the `change` parameter onto the current stock price.
     * Re-render the view so that the new price is displayed to the user.
-1. The application entry-point (`src/app.js`) should:
-  * Implement the anonymous function passed to `setInterval`. It should:
-    * Get the list of all `QuoteView` instances from the `ApplicationView`.
-    * Call the `simulate` function on each of those `QuoteView` instances.
+1. Something in the project should call the `simulate` function on each of the `QuoteView` instances.
```

Comment:
Maybe have them implement simulate as a method on the model and have the view call that method every x seconds.

Reply:
I would expect that `simulate` would move out of `app.js`, but I'll reword it so it's more explicit.

Reply:
I don't think that the view should call the `simulate` method at all, since that is not an event coming from the user / DOM.

If we put it in the model, then the model itself should have the `setTimeout` code to run the simulation once per second.

Reply:
I could see all of this logic in the model, that could work.

But also, if I think about this as "how would this be placed in an app," I would honestly think that there should be something that's not a view and not a model triggering some event. In that case, then the view should be listening to that event and updating the model (even if it's calling a custom method defined on that model to update it).

(I understand that this logic is contradictory to my proposed changes README haha)